docs/TESTING.md

```diff
@@ -36,22 +28,61 @@ We're supporting Android and iOS mobile operating systems.
 
 #### Scope
 
-In manual testing, we should focus on all user interface elements, hand washing animation in "Protect Yourself".
+In manual testing, we should focus on all user interface interactions:
+
+* Protect yourself
+* Your questions answered
+* Individual questions
+* News & press
+* External links w/titles/brief text
+* Travel advice
+* Myth-busters
```

Comment:
This feature doesn't exist.

Reply:
@advayDev1 I've taken it from Design Doc for v1.0, we don't want to implement it no more?